src/FusionExport/ExportManager.php

```diff
@@ -72,20 +77,26 @@ public function exportAsStream(ExportConfig $exportConfig, $exportBulk=true) {
 			$tempDir = tempnam('tmp', 'fusioncharts');
 			$fileName = $tempDir . '_export.zip';
 
+            
+
 			file_put_contents($fileName, $contents);
-			$zipFile = new \ZipArchive();
+			$zipFile = new ZipArchive;
```

Comment:
is this intentional?

Reply:
new ZipArchive without parenthesis

Reply:
My Bad, not an intentionally. Added Parenthesise.

Reply:
Can you please run this functionality and check one please. Thanks.

Reply:
Hi
i checked, files are exporting and working fine.